### ISSUE TITLE
profile.rb:258:in `block in get_company_details': undefined method `[]' for nil:NilClass (NoMethodError)

### ISSUE THREAD

**Issue by bricemaurin** [yatish27/linkedin-scraper#86]
I get this error when trying to get linkedin data

````
/usr/local/var/rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/linkedin-scraper-2.1.0/lib/linkedin-scraper/profile.rb:258:in `block in get_company_details': undefined method `[]' for nil:NilClass (NoMethodError)
````

**Comment by yatish27** [comment]
looking into it.

**Comment by yatish27** [comment]
@bricemaurin Fixed check 2.1.1 and master

**Comment by bricemaurin** [comment]
Thanks ;)
